src/components/WebexVideoSettings/WebexVideoSettings.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from 'react';
+import {
+  Select, SelectOption,
+} from '@momentum-ui/react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+import WebexLocalMedia from '../WebexLocalMedia/WebexLocalMedia';
+
+/**
+ *
```

Comment:
put the description of the function first "Returns all of available cameras"
and a minimal message in the return, but with the correct type (array):
```js
* @returns {object[]} Available cameras
* ```